kernel_gateway/proposal.md

```diff
@@ -0,0 +1,90 @@
+# Proposal for Incubation
+
+## Subproject name
+
+Jupyter Kernel Gateway (`jupyter-incubator/kernel_gateway`)
+
+## Development team and Advocate
+
+* Peter Parente, IBM (`@parente`)
+* Dan Gisolfi, IBM (`@vinomaster`)
+* Justin Tyberg, IBM (`@jtyberg`)
+* Gino Bustelo, IBM (`@ginobustelo`)
+
+Who is the Steering Council Advocate for this Subproject?
+
+* Kyle Kelley (`@rgbkrk`)
+
+## Subproject goals, scope and functionality
+
+### Problem
+
+Applications that use Jupyter kernels as execution engines outside of the traditional notebook / console user experience have started appearing on the web (e.g., [pyxie](https://github.com/oreillymedia/pyxie-static), [pyxie kernel server](https://github.com/oreillymedia/ipython-kernel), [Thebe](https://github.com/oreillymedia/thebe), [gist exec](https://github.com/rgbkrk/gistexec), [notebooks-to-dashboards](http://blog.ibmjstart.net/2015/08/22/dynamic-dashboards-from-jupyter-notebooks/), ). Today, these projects rely on:
```

Comment:
The nice thing about the notebook and jupyterhub is that they can connect directly to remote kernels in the same way, there's just not code to reach out to providers/gateways. I personally think we can build out the gateway before pushing any of this vision on the main notebook though, as it gives us a smaller scope to iterate on.